src/_forms.scss

```diff
@@ -60,239 +150,306 @@ category: Common
     }
+}
 
-    .form-fields__heading {
-        @include fs-bodyHeading(3, true);
-        margin: 0 0 $gs-baseline;
-        display: block;
+.form-note--right {
+    @include mq(tablet) {
+        text-align: right;
+        margin-left: rem($gs-gutter);
     }
 }
 
-.form-fields--date-of-birth .form-field {
-    margin: 0 18px 0 0;
+.form-note--underneath {
+    margin-top: rem($gs-baseline / 2);
+}
+
+.form-legal {
+    @include fs-textSans(3);
+    color: $pasteup-forms-note-colour;
 }
 
+// Form labels and inputs
+// ============================================================================
+
 .label {
+    @include fs-bodyCopy(2);
+    color: $pasteup-forms-label-colour;
     cursor: pointer;
     display: block;
-    margin-bottom: 6px;
 }
 
-.text-input,
-.textarea {
-    border: 1px solid $pasteup-forms-fields-border-colour;
-    @include box-shadow(none);
-    @include box-sizing(border-box);
-    color: $pasteup-forms-fields-colour;
+.label--inline {
     display: inline-block;
-    font-family: $pasteup-forms-fields-font-family;
-    padding: 8px 8px 7px;
-    font-size: 16px;
-    line-height: 1.4;
+}
+
+.input-text,
+.input-textarea {
+    color: $pasteup-forms-input-colour;
+    border: 1px solid $pasteup-forms-input-border-colour;
+    @include f-textSans;
+    @include font-size(16px, 22px);
+    padding: rem(8px) rem(8px) rem(7px);
     outline: none;
-    @include border-radius(0);
+    @include box-sizing(border-box);
     width: 100%;
     -webkit-appearance: none;
 
     &:focus {
-        border-color: $pasteup-forms-fields-border-colour-focus;
+        border-color: $pasteup-forms-input-border-colour-focus;
     }
 
     @include mq(tablet) {
-        font-size: 14px;
+        font-size: rem(14px);
     }
 }
 
-.textarea {
+.input-text--small {
+    width: 30%;
+}
+
+.input-textarea {
     resize: vertical;
 }
 
-.textarea--no-resize {
-    min-height: ($gs-baseline/3)*20;
-    resize: none;
+.input-textarea--long {
+    min-height: rem(($gs-baseline / 3) * 40);
 }
 
-.textarea--mid {
-    min-height: $gs-baseline*9;
+.optional-marker {
+    &:after {
+        vertical-align: middle;
+        content: '(optional)';
+        color: $pasteup-forms-note-colour;
+        @include fs-textSans(1);
+        margin-left: rem(4px);
+    }
 }
 
-.textarea--long {
-    min-height: ($gs-baseline/3)*40;
+.select {
+    font: inherit;
 }
 
-.submit-input {
-    background: $pasteup-forms-submit-background;
-    border: 0 none;
-    color: #ffffff;
-    cursor: pointer;
-    display: inline-block;
-    font-size: 14px;
-    margin: 0 $gs-gutter 0 0;
-    min-width: gs-span(2);
-    padding: 11px $gs-gutter/2;
-    outline: none;
-    text-align: center;
+// width needed for iphone so select doesn't exceed screen width
+.select--wide {
+    width: 100%;
+}
 
-    &:hover,
-    &:focus {
-        background: $pasteup-forms-submit-background-focus;
+.select--minimal {
+    border: 0;
+    outline: 0;
+    background-color: #ffffff;
```

Comment:
Does this need to be shared? If so, probably should be `transparent` so we're not assuming page background colour.